src/helpers.js

```diff
@@ -60,6 +60,9 @@ export function eachOperation(spec, cb, find) {
   // Iterate over the spec, collecting operations
   for (const pathName in paths) {
     for (const method in paths[pathName]) {
+      if (method.toUpperCase() === 'PARAMETERS') { 
```

Comment:
Oddly, when path-parameters come before the operation like below, the method becomes `PARAMETERS` (here it should be `GET`), and that causes problems.
```
paths: {
  '/pet/{id}': {
    parameters: [
      {...}
    ],
    get: {
      operationId: 'getPetsById',
      parameters: [
        {...}
      ],
    }
  }
}
```